library/src/skins/default/src/sass/modules/tool/portal/_portal.scss

```
@@ -130,22 +130,15 @@
   }
 }
 
+.role-switcher {
+    margin-right: 5px;
```

To align with the `link-button`
```suggestion
    margin: 2px 5px;
```